dali/test/python/test_operator_audio_resample.py

```diff
@@ -34,64 +34,76 @@
 rates = [ 16000, 22050, 12347 ]
 lengths = [ 10000, 54321, 12345 ]
 
-def create_test_files():
```

Comment:
Note: nose was picking this as a test

Reply:
Making it "private" (with leading underscore) would help, too.